src/backend/distributed/executor/adaptive_executor.c

```diff
@@ -242,10 +242,14 @@ typedef struct DistributedExecution
 	bool raiseInterrupts;
 
 	/*
-	 * Flag to indicate whether the query is running in a distributed
-	 * transaction.
+	 * useRemoteTransactionBlocks is used to manage remote transactions according
+	 * to one of the following policies:
+	 *
+	 * - opening a remote transaction is required
```

Comment:
super minor: The order of the comments doesn't match with the order in the enum definition. So, maybe fix the order or add the enum name in parenthesis here?

(it confused me at first)